From c390e5f2d0521d6d10b113ea13b629804e80f2f2 Mon Sep 17 00:00:00 2001
From: RagedMeteor1837 <email@pingushrek.co.uk>
Date: Thu, 08 May 2025 16:54:18 +0100
Subject: [PATCH] Change bug report URL


diff --git a/src/main/java/com/mojang/launcher/game/runner/AbstractGameRunner.java b/src/main/java/com/mojang/launcher/game/runner/AbstractGameRunner.java
index cd37a5b..d9c609c 100644
--- a/src/main/java/com/mojang/launcher/game/runner/AbstractGameRunner.java
+++ b/src/main/java/com/mojang/launcher/game/runner/AbstractGameRunner.java
@@ -291,7 +291,7 @@ public abstract class AbstractGameRunner implements GameRunner, DownloadListener
                         this.setStatus(GameInstanceStatus.LAUNCHING);
                         this.launchGame();
                     } catch (Throwable var5) {
-                        LOGGER.fatal("Fatal error launching game. Report this to http://bugs.mojang.com please!", var5);
+                        LOGGER.fatal("Fatal error launching game. Report this to https://github.com/RagedMeteor1837/mclauncher/issues/new please!", var5); // olauncher - have people report bugs to us
                     }
                 }
             }
-- 
2.40.1

